tests/generic_relations/tests.py

```diff
@@ -513,6 +514,55 @@ def test_subclasses_with_parent_gen_rel(self):
         TaggedItem.objects.create(content_object=bear, tag='orange')
         self.assertEqual(Carrot.objects.get(tags__tag='orange'), bear)
 
+    def test_generic_inline_formsets_initial_count(self):
```

Comment:
Please move these tests to `test_forms.py` following bcadabe38afd3fc46c9e821c68cccd7ca0c30129.